recipes/catch2/2.x.x/conanfile.py

```diff
@@ -51,79 +42,80 @@ def config_options(self):
 
     def configure(self):
         if not self.options.with_main:
-            del self.options.fPIC
-            del self.options.with_benchmark
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+            try:
+                del self.options.with_benchmark
+            except Exception:
+                pass
 
     def package_id(self):
         if not self.options.with_main:
             self.info.header_only()
```

Comment:
```suggestion
            self.info.clear()
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-package-id-method

Reply:
Yes actually I did know that, I did it in a diff recipe, that one slipped through, thanks!